src/requirements/Allowlist/AllowlistForm.tsx

```diff
@@ -43,16 +82,40 @@ const AllowlistForm = ({ baseFieldPath }: RequirementFormProps): JSX.Element =>
         ?.toString()
         ?.split("\n")
         ?.filter((line) => !!line)
-        ?.map((line) => line.slice(0, 42))
+        ?.map((line) =>
+          type === "ALLOWLIST_EMAIL" ? line.trim() : line.slice(0, 42)
```

Comment:
I think this will "break" Fuel addresses (those are 66 long incl. 0x afaik) :ohno:

Reply:
I'm not sure if it currently works on prod then :D Should we just .trim always?

Reply:
Oh 😂  Yeah, then let's just trim them 👍

Reply:
Made the change in https://github.com/guildxyz/guild.xyz/pull/1135/commits/7543acfad1b80896fb5735a0d7b6d9a7851031b2 👍